linkcheck/director/aggregator.py

```diff
@@ -18,7 +18,10 @@
 Aggregate needed object instances for checker threads.
 """
 import threading
-import thread
+try: # Python 3
+    import _thread as thread
+except ImportError:
+    import thread
```

Comment:
In other modules you do `import thread as _thread` on Python 2.  I think I prefer that.